TraderJoe-Bots/Large-Swap/src/agent.spec.ts

```diff
@@ -0,0 +1,512 @@
+import { FindingType, FindingSeverity, Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { TestTransactionEvent, createAddress, MockEthersProvider, MockEthersSigner } from "forta-agent-tools/lib/tests";
```

Comment:
```suggestion
import { TestTransactionEvent, createAddress, MockEthersProvider } from "forta-agent-tools/lib/tests";
```
Unused imported module.